tests/scripts/app_command.lua

```diff
@@ -99,6 +99,21 @@ do -- ExportSpriteSheet
     0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,
     0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,
   })
+
+  local s = Sprite{ fromFile="sprites/groups4.aseprite" }
+  local c = app.pixelColor.rgba(75, 105, 47)
+  app.command.ExportSpriteSheet {
+    type=SpriteSheetType.ROWS,
+    textureFilename="_test_export_spritesheet4.png",
+    layer="G1/G2/G3/b",
+    trim=true,
+  }
+  local i = Image{ fromFile="_test_export_spritesheet4.png" }
+  expect_img(i,  {
+    c,c,0,
+    c,c,0,
+    c,c,c,
+  })
```

Comment:
I think we can use `groups3abc.aseprite` and `groups4.aseprite` from this patch:
```suggestion

  local s = Sprite{ fromFile="sprites/groups3abc.aseprite" }
  local c = app.pixelColor.rgba(75, 105, 47)
  app.command.ExportSpriteSheet {
    type=SpriteSheetType.ROWS,
    textureFilename="_test_export_spritesheet4.png",
    layer="b/b",
    trim=true,
  }
  local i = Image{ fromFile="_test_export_spritesheet4.png" }
  expect_img(i,  {
    c,c,0,
    c,c,0,
    c,c,c,
  })
```